### DISCUSSION TITLE
JavaScript File Render Bug

### DISCUSSION THREAD

**Discussion by jumanji144** [community/community#63366] [Repositories]
### Select Topic Area

Bug

## Observation
 Github code view fails to correctly render javascript strings with emojis in them

### Expected
The string `"Made with ❤️ hello test 123! ❤️ heart",` to be displayed correctly

### Actual
![image](https://github.com/community/community/assets/55301990/f60ac53e-fd55-45ba-849b-5d2450a3abdb)

## Steps to reproduce
- In any repository create a new javascript file
- Enter  `"Made with ❤️ hello test 123! ❤️ heart",`
- Observe behaviour

**Comment by robskaar** [comment]
Hi, this is not reproduceable, please give more information about your environment if not fixed already.
i tried in a github code space and in a github repository, and both was fine.

![billede](https://github.com/community/community/assets/31418455/858bcd0d-6f78-4383-ab22-14b77d0b464d)

**Comment by jumanji144** [comment]
After further review it seems that it has to do with the unicode renderer of my browser (firefox) which caused the original issue. I originally tested the same content on other sites and it behaved fine there. I guess firefox misinterprets the DOM on the github website.

**Reply by robskaar** [reply]
Thanks for your feedback, Ill try one of the next days to reproduce in firefox, different versions, firefox developer & incognito, see where this happens and also see if i can find documentation to link to this thread describing the issue to future readers 👍🏼

**Reply by jumanji144** [reply]
It does appear though that the bug has lessened, it only happens now when trying to select the text and it also randomly switches between the github rendered emojis and standard unicode font emoji for the heart
![image](https://github.com/community/community/assets/55301990/7deb322d-5dd4-4a38-954a-dbbb2bb8850c)
![image](https://github.com/community/community/assets/55301990/18e17491-9796-4589-b124-42f8ecb875b2)
